lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,15 +1,7 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
 
 module SemanticPuppet
-  if defined?(GettextSetup)
-    GettextSetup.initialize(File.absolute_path('semantic_puppet/locales', File.dirname(__FILE__)))
-  end
```

Comment:
If so, then we need to change the original semantic_puppet gem and release a 1.0.1 version of it. But that, in turn, gets awkward since the semantic_puppet gem has a hard dependency on gettext. Making that a loose dependency is complicated since semantic_puppet has no knowledge of features.